plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek.hs

```diff
@@ -141,4 +141,7 @@ readKnownCek
     => MachineParameters CekMachineCosts CekValue uni fun
     -> Term Name uni fun ()
     -> Either (CekEvaluationException uni fun) a
-readKnownCek params = evaluateCekNoEmit params >=> readKnown
+readKnownCek params t = do
```

Comment:
i think this readknowncek are used only in testing. what if we maded the KnownType class work on ETerm's directly?